realm/realm-library/src/androidTest/kotlin/io/realm/ManagedSetTester.kt

```diff
@@ -162,8 +163,18 @@ class ManagedSetTester<T : Any>(
                 set.add(value)
```

Comment:
I guess this works, but it could be dangerous to only test remove on a single element set.